kubetest/anywhere.go

```diff
@@ -34,6 +34,7 @@ var (
 	kubernetesAnywhereKubeadmVersion = flag.String("kubernetes-anywhere-kubeadm-version", "stable", "(kubernetes-anywhere only) Version of kubeadm to use, if phase2-provider is kubeadm. May be \"stable\" or a gs:// link to a custom build.")
 	kubernetesAnywhereCluster        = flag.String("kubernetes-anywhere-cluster", "", "(kubernetes-anywhere only) Cluster name. Must be set for kubernetes-anywhere.")
 	kubernetesAnywhereUpTimeout      = flag.Duration("kubernetes-anywhere-up-timeout", 20*time.Minute, "(kubernetes-anywhere only) Time limit between starting a cluster and making a successful call to the Kubernetes API.")
+	kubernetesAnywhereCNIName        = flag.String("kubernetes-anywhere-cni-name", "weave", "(kubernetes-anywhere only) The name of the CNI plugin used for the cluster's SDN.")
```

Comment:
Nitpick: since you've dropped the separate config string, I would just drop the `-name` suffix across the board for brevity.

`s/cni[-_]name/cni/g`
`s/CNIName/CNI/g`

We similarly use just the word `cluster` instead of `cluster-name`.